server/cards/cards.controller.js

```diff
@@ -0,0 +1,22 @@
+import bodyParser from "body-parser";
+import { Router } from "express";
+import Cards from "./cards.model";
+
+const router = Router();
+
+router.route("/").post(bodyParser.json(), async (request, response) => {
+  try {
+    const cards = new Cards(request.body);
+    await cards.save();
+    return response.status(200).json("Card created!");
```

Comment:
This seems like a very naive solution to a complex problem. Lets have a chat at tomorrow's standup or async after it to discuss what needs to be done


cc @gargi-bansal @michellemtan

Reply:
Yeah in retrospect i think i should've had a discussion with the group on how we expect to add new cards and data in.